addons/cargo/functions/fnc_handleDestroyed.sqf

```diff
@@ -24,9 +24,9 @@ if (_loaded isEqualTo []) exitWith {};
 {
     // TODO Do we want to be able to recover destroyed equipment?
     if (_x isEqualType objNull) then {
+        detach _x;
         deleteVehicle _x;
```

Comment:
Will `setDamage` trigger killed EHs? If so I wonder if we should kill the cargo so `killed` handlers will fire on it and delete it frame later?

Reply:
It should however, I think it's unnecessary for us to do it here.